### PR TITLE
[CUBRIDMAN-211] Manual for Enhance OPTIMIZER

### PR DESCRIPTION
**Pull request by shparkcubrid**
http://jira.cubrid.org/browse/CUBRIDMAN-211

- Number of sampling statistical information pages changed from 7 to 5000.
- Add NDV when executing ';info stats table' command.
- SELECT authorization is required when updating statistical information.
- Add fetch time in trace information.
- Add leading SQL hint.

### CODE REVIEW

**Comment by hiclass** [comment]
추가된 LEADING 힌트의 예제와 ORDERD 예제를 같이 추가해서 사용자에게 차이점을 알려주었으면 합니다.

File: ko/sql/tuning.rst [hunk 1]
```diff
@@ -763,6 +772,27 @@ SQL 힌트는 주석에 더하기 기호(+)를 함께 사용하여 지정한다.
 
     <*spec_name*>\ 이 주어지지 않고 **USE_NL**\ 과 **USE_MERGE**\ 가 함께 지정된 경우 주어진 힌트는 무시된다. 일부 경우에 질의 최적화기는 주어진 힌트에 따라 질의 실행 계획을 만들지 못할 수 있다. 예를 들어 오른쪽 외부 조인에 대해 **USE_NL**\ 을 지정한 경우 이 질의는 내부적으로 왼쪽 외부 조인 질의로 변환이 되어 조인 순서는 보장되지 않을 수 있다.
 
+.. note::
+
+    **ORDERED**이 **LEADING**과 함께 지정될 경우 **LEADING** 힌트는 무시된다.
+    **LEADING**힌트가 여러게 지정될 경우 첫번째 **LEADING** 힌트만 적용된다.
```
Comment (youngjinj) [inline ko/sql/tuning.rst]:
```suggestion
    **ORDERED**\가 **LEADING**\과 함께 지정될 경우 **LEADING** 힌트는 무시된다.
    **LEADING** 힌트가 여러게 지정될 경우 첫번째 **LEADING** 힌트만 적용된다.
```

File: ko/sql/tuning.rst [hunk 2]
```diff
@@ -763,6 +772,27 @@ SQL 힌트는 주석에 더하기 기호(+)를 함께 사용하여 지정한다.
...
+
+    .. code-block:: sql
+
+        SELECT /*+ ORDERED LEADING(b, d) */ *
+        FROM a INNER JOIN b ON a.col=b.col
+        INNER JOIN c ON b.col=c.col INNER JOIN d ON c.col=d.col;
+
+    위와 같은 질의를 수행한다면 **LEADING** 힌트는 무시되며, **ORDERED** 힌트에 따라서 **FROM**절의 순서인, 테이블 a, b, c, d의 순서로 조인된다.
```
Comment (youngjinj) [inline ko/sql/tuning.rst:786]:
```suggestion
    위와 같은 질의를 수행한다면 **LEADING** 힌트는 무시되며, **ORDERED** 힌트에 따라서 **FROM** 절의 순서인, 테이블 a, b, c, d의 순서로 조인된다.
```

File: ko/sql/tuning.rst (hunk 1 above)
Comment (youngjinj) [reply ko/sql/tuning.rst]: Please change to resolve WARNING.

File: ko/sql/tuning.rst (hunk 2 above)
Comment (youngjinj) [reply ko/sql/tuning.rst:786]: Please change to resolve WARNING.

**Review by youngjinj — APPROVED** [review]

**Review by sungc-cubrid — APPROVED** [review]

**Review by bagus-kim — APPROVED** [review]

File: ko/sql/tuning.rst [hunk 3]
```diff
@@ -98,18 +102,18 @@ CSQL 인터프리터의 세션 명령어로 지정한 테이블의 통계 정보
...
     ****************
-     Class name: t1 Timestamp: Mon Mar 14 16:26:40 2011
+     Class name: t1 Timestamp: Mon Mar 25 17:56:10 2024
      Total pages in class heap: 1
      Total objects: 5
      Number of attributes: 1
-     Attribute: code
-        id: 0
-        Type: DB_TYPE_INTEGER
-        Minimum value: 1
```
Comment (hiclass) [inline ko/sql/tuning.rst:108]: Wasn't the Min/MAX value the only one without a reason?

**Review by kwonhoil — APPROVED** [review]

**Review by hiclass — APPROVED** [review]

File: ko/sql/tuning.rst (hunk 3 above)
Comment (shparkcubrid) [reply ko/sql/tuning.rst:108]: It has been removed a long time ago from the JIRA below.
http://jira.cubrid.org/browse/CUBRIDSUS-11973

I think min/max values are necessary for selectivity of range operators etc.

**Review by xmilex-git — APPROVED** [review]

**Review by mhoh3963 — APPROVED** [review]

**Pull request merged**